server/meca/routes.js

```diff
@@ -0,0 +1,39 @@
+const logger = require('@pubsweet/logger')
+const config = require('config')
+// TODO avoid this recursive dependency
```

Comment:
It's worth putting this TODO into an issue.

Incorporating pubsweet's full data model implementation should solve this.

Reply:
See #708